File: 1011 - volume_esfera/esfera.go
package main

import "fmt"

func vesfera(r float64) float64 {
	return ((4.0 / 3.0) * 3.14159 * r * r * r)
}

func main() {
	var r, vol float64
	fmt.Scanf("%f\n", &r)
	vol = vesfera(r)
	fmt.Printf("VOLUME = %.3f\n", vol)
}

File: 1011 - volume_esfera/esfera.cpp
#include <iostream>
#include <iomanip>

using namespace std;

double vesfera( double r )
{
    return (( 4.0 / 3.0 ) * 3.14159 * r * r * r );
}

int main()
{
    double r,vol;
    cin >> r;
    vol = vesfera (r);
    cout << "VOLUME = " << fixed << setprecision(3) << vol << "\n";
    return 0;
}